BankSDK/GiniBankSDKExample/GiniBankSDKExample/Screen API/ScreenAPICoordinator.swift

```diff
@@ -139,12 +139,13 @@ final class ScreenAPICoordinator: NSObject, Coordinator, UINavigationControllerD
             }
 		}
 	
-		configuration.cleanup(paymentRecipient: extractedResults.first(where: { $0.name == "paymentRecipient"})?.value ?? "",
+		configuration.sendTransferSummary(paymentRecipient: extractedResults.first(where: { $0.name == "paymentRecipient"})?.value ?? "",
                               paymentReference: extractedResults.first(where: { $0.name == "paymentReference"})?.value ?? "",
                               paymentPurpose: extractedResults.first(where: { $0.name == "paymentPurpose"})?.value ?? "",
                               iban: extractedResults.first(where: { $0.name == "iban"})?.value ?? "",
                               bic: extractedResults.first(where: { $0.name == "bic"})?.value ?? "",
                               amountToPay: extractionAmount)
+        configuration.cleanup()
```

Comment:
[suggestion]: Indentation issue `configuration.sendTransferSummary`.
Please check also the indentation in this method `closeSreenAPIAndSendFeedback`.